index.js

```diff
@@ -0,0 +1,150 @@
+// ЗАДАЧИ логические операторы
+
+// Проверка значения из диапазона
+
+if (age >= 14 && age <= 90);
+
+// Проверка значения вне диапазона
+if (!(age >= 14 && age <= 90));
+
+if (age < 14 || age > 90);
+
+// ПРОВЕРКА ЛОГИНА
+let login = prompt("Кто там?", "");
+if (login == "Admin") {
+  console.log("");
+  let password = prompt("пароль?", "");
+  if (password == "я главный") {
+    console.log("Здравствуйте");
+  } else if (password == null) {
+    console.log("отменено");
+  } else {
+    console.log("неверный пароль");
+  }
+} else if (login == null) {
+  console.log("Отменено");
+} else {
+  console.log("я вас не знаю");
+}
+
+// 1)Если переменная a больше нуля и меньше 5-ти, то выведите 'Верно', иначе выведите 'Неверно'. Проверьте работу скрипта при a, равном 5, 0, -3, 2.
+
+let a = prompt("введите число", "");
+if (a > 0 && a < 5) {
+  console.log("верно");
+} else {
+  console.log("неверно");
+}
+// при а = 2 будет верно, при остальных значениях будет неверно.
+
+//  2) Если переменная a равна нулю или равна двум, то прибавьте к ней 7, иначе поделите ее на 10.
+//  Выведите новое значение переменной на экран.
+// Проверьте работу скрипта при a, равном 5, 0, -3, 2.
+
+let f = prompt("число", "");
+if (f == 0 || f == 2) {
```

Comment:
remove checking for 0

and use strict comparison

Reply:
fixed